ci-operator/step-registry/ipi/conf/openstack/techpreview/ipi-conf-openstack-techpreview-chain.yaml

```diff
@@ -0,0 +1,19 @@
+chain:
+  as: ipi-conf-openstack-techpreview
+  steps:
+  - ref: openstack-conf-clouds
+  - ref: openstack-conf-resourcenames
+  - ref: openstack-conf-createfips
+  - ref: openstack-conf-generateconfig
+  - ref: openstack-conf-creatednsrecords
+  - ref: openstack-conf-exportcindercredentials
+  - ref: ipi-conf-etcd-on-ramfs
+  - ref: ipi-conf-openstack-precheck
+  - ref: ipi-install-monitoringpvc
```

Comment:
I feel like there should be a chain gathering up some of this, to reduce the chance that someone adds a new OpenStack knob similar to the existing FIPS, but forgets to update this tech-preview chain to respect that knob here.

Reply:
There doesn't appear to be anything today that I can find that resembles a shared chain for this stuff, have poked the ShiftStack team regarding the issue to see if this is something on their radar

Reply:
@pierreprinetti suggested that we can actually avoid this duplcation by not using the ipi-conf like this and just directly adding the techpreview step into the workflow. Since the AWS version of this is already merged, https://github.com/openshift/release/pull/21227 is a PR which cleans it up to show what Pierre meant, do you think this is a better approach that I should adopt for all of these?